package.json

```diff
@@ -70,7 +70,7 @@
     "angular2-template-loader": "0.6.2",
     "async": "2.6.3",
     "awesome-typescript-loader": "5.2.1",
-    "chromedriver-version-matcher": "^1.0.0-alpha.2",
+    "chromedriver-version-matcher": "^1.0.0-alpha.6",
```

Comment:
I believe the caret (`^`) only picks up "latest" versions; so it might be better to write this as `>1.0.0-alpha` so that we get the most recent "next" package for each build.

Reply:
I think using `>` wouldn't limit us to the `1.X.X` major version though.

Reply:
Ah, good point. You could do `>1.0.0-alpha < 2.0.0`.

Reply:
If the `chromedriver-version-matcher` were released as a major version (and not a pre-release), would your suggestion be the same?

It feels like we're having to deviate from our normal with the version requirements only because that library isn't officially released.

If that's the case, I would almost rather release that library as a non-pre-release (the API hasn't changed since it's release) and leave this requirement as the caret.

Thoughts?

Reply:
If we released a major version, my suggestion would still work, installing the "latest" version in that range. You can see an example of this using the semver calculator, here:
https://semver.npmjs.com/
Using the default `lodash` library, type in the following range: `>1.0.0-rc < 2.0.0`. (I'm using the same range with the Angular 9 branch as well.)

That being said, it would definitely be "cleaner" to release a major version of the library and just use the caret `^`.

Reply:
Am I understanding correctly, you're wanting to write this such that the version tagged with `@next` is pulled in?  

Aside from the wonkiness with this library still being alpha, I don't think that's the behavior we want.

Reply:
https://github.com/blackbaud/chromedriver-version-matcher/pull/18